### PR TITLE
Chore: Fix typing when integrate with other project

### PR DESCRIPTION
**Pull request by ttkien**
Fix typings

### CODE REVIEW

File: lib/src/bindings/Helpers.gen.ts [hunk 1]
```diff
@@ -4,26 +4,32 @@
...
-exactType(Hero_Theme, Legacy_Theme); // error
+/*eslint-disable-next-line */
+let exactType:
+  | (<T, U>(
+      draft: T & IfEquals<T, U>,
+      expected: U & IfEquals<T, U>,
+    ) => IfEquals<T, U>)
+  | undefined;
+
+if (exactType) {
```
Comment (ttkien) [inline lib/src/bindings/Helpers.gen.ts:26]: check null because crashed happen

File: lib/src/components/ListItem.re [hunk 2]
```diff
@@ -18,24 +18,24 @@ let emptyStyle = Style.style();
 [@react.component]
 let make =
     (
-      ~title,
-      ~subtitle=?,
+      ~title: string,
+      ~subtitle: option(string)=?,
```
Comment (ttkien) [inline lib/src/components/ListItem.re:22]: Specify type is `string`

File: lib/src/shims/ReactNative.shim.ts [hunk 3]
```diff
@@ -5,9 +5,15 @@ import {
   TextInputKeyPressEventData,
   ImageSourcePropType,
   ViewStyle,
+  StyleProp,
+  TextStyle,
+  ImageStyle,
 } from 'react-native';
 
-export type Style_t = ViewStyle;
+export type Style_t =
```
Comment (ttkien) [inline lib/src/shims/ReactNative.shim.ts:13]: fix typing

File: lib/src/components/ListItem.re (hunk 2 above)
Comment (toan2406) [reply lib/src/components/ListItem.re:22]: Do we need to check if mobile only use string in title? Title is react element is intentional, as user can pass <Text> with custom style to title

File: lib/src/components/TouchableTextInput.re [hunk 4]
```diff
@@ -28,7 +28,7 @@ type keyboardType = [
 [@react.component]
 let make =
     (
-      ~touchableTestID=?,
+      ~testID=?,
```
Comment (toan2406) [inline lib/src/components/TouchableTextInput.re:31]: Why need to change this? is it breaking change?

File: lib/src/components/ListItem.re (hunk 2 above)
Comment (ttkien) [reply lib/src/components/ListItem.re:22]: https://github.com/Thinkei/rn-hero-design/blob/2ff2ef79286d2e41b12bfcbb6feff9d51008ab46/lib/src/components/ListItem.re#L43

I found title and subtitle is a value of `Text` component

**Pull request merged**